tech.json

```diff
@@ -801,7 +801,21 @@
             "Used to fall on top of breakable bomb blocks or enemies.",
             "Performed by storing a spark while falling off a ledge, holding angle and crouching after storing a spark, or unmorphing."
           ],
-          "devNote": "This is not a momentum continuing action like canBlueSpaceJump"
+          "devNote": "This is not a momentum continuing action like canBlueSpaceJump",
+          "extensionTechs": [
+            {
+              "name": "canSpeedKeepChain",
+              "requires": [
+                "canSpeedKeep",
+                "canLateralMidAirMorph"
+              ],
+              "note": [
+                "The ability to chain multiple speedkeeps in a row. This can be done by jumping and doing a stationary midair morph on the descent, then soft unmorph, then holding an angle during and after landing.",
```

Comment:
Stationary midair morph sounds similar to `canStationaryLateralMidAirMorph` and this isn't that.

The tech should mention aim down pose being safe and you need to transition between angle to down.  That detail can be tricky.

"holding an angle during ... landing" would prevent morphing so you have to release it.  But it is true that during the unmorph angle should be held.  The way its written can be misunderstood.